src/dex/docs.rb

```diff
@@ -17,27 +17,32 @@ class LookupFail < Exception
   Reply = Struct.new(:content, :embed)
 
   # Module for helpers for building embeds
-  # TODO: Also GitHub permalinks?
   module Embed
     # For building permalinks
     RUBYDOC = 'http://www.rubydoc.info/github/meew0/discordrb/master'
 
     # Does this really need explaining?
     RUBY_TACO = 'https://cdn.discordapp.com/emojis/315242245274075157.png'
 
+    # Git Version of the library
+    GIT_VERSION = Bundler.load.specs['discordrb'].first.git_version.strip
+
+    # Base link for the library source
+    GITHUB_URL = 'https://github.com/meew0/discordrb'
+
     # Utility method that yields a template embed
     def new_embed
       definitions = files.map do |f|
         path, line = f[0].split('/')[8..-1].join('/'), f[1]
-        "`#{path}` `L#{line}`"
+        "`#{path}`#`L#{line}`"
```

Comment:
No, just remove the code block ticks from here and interpolate it into the mask in `description` below so you don't have to `.tr`